app/src/main/java/io/pumpkinz/pumpkinreader/model/Comment.java

```diff
@@ -13,13 +15,17 @@
     Comment parentComment;
     List<Comment> childComments;
 
-    public Comment() {}
+    public Comment() {
+        this.kids = new ArrayList<>();
+        this.childComments = new ArrayList<>();
```

Comment:
Is it the rule of Retrofit to match the field name?
If it's not, better be changed.